modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="determining-upgrade-viability-conditiontype_{context}"]
+= Determining upgrade viability with conditionType
+
+The status of cluster operators is revealed with the value of conditionType, informing you of potential impacts to upgrading your cluster. 
+
+* Available: 
+The conditionType `Available` indicates that the operators are functional and available in the cluster. If the status is `Available=False`, then the status indicates that at least one part of the operand is non-functional and the condition requires an administrator to intervene.
+
+* Progressing:
+The conditionType `Progressing` indicates that the operator is actively rolling out new code, propagating configuration changes, or otherwise moving from one steady state to another. Operators should not report as `Progressing` when they are reconciling a previous known state. If the observed cluster state has changed and the operator is reacting to it, then the status should report back as `Progressing=True`, since it is moving from one steady state to another.
+
+* Degraded:
+A `Degraded` conditionType indicates that the operator's current state does not match its desired state over a period of time. The period of time may vary by component, but a `Degraded` state represents persistent observation of a condition.  As a result, a component should not fluctuate in and out of the `Degraded` state.  
++
+A service may have the conditionType `Available`, even if it is degraded. This indicates that the service is `Available` but `Degraded` because it may have a lower quality of service. 
++
+There may be a different conditionType if the transition from one state to another does not persist over a long enough period to report `Degraded`.  
```

Comment:
Maybe we can drop this line?  I'm having a hard time imagining how reading it will help a condition consumer with a particular action.  Perhaps a generic statement at the top of the section something like:

> The following condition types have consistent semantics across all ClusterVersion and ClusterOperator conditions.  Individual components may also have additional condition types with component-specific semantics.

to explain that the list of condition types is not exhaustive?